cfy_manager/components/restservice/restservice.py

```diff
@@ -271,8 +273,12 @@ def _configure():
     set_logrotate(RESTSERVICE)
     _deploy_sudo_commands()
     _configure_restservice()
+    if config[CLEAN_DB]:
+        db.prepare_db()
     systemd.configure(RESTSERVICE, tmpfiles=True)
     _start_restservice()
+    if config[CLEAN_DB]:
+        db.populate_db()
```

Comment:
does this actually need to be run after the restservice is *started*? the `create-tables.py` script shouldn't depend on that

Reply:
Yup, it no longer does